source/adapter.screenshare.js

```diff
@@ -0,0 +1,161 @@
+(function () {
+
+  'use strict';
+
+  var tempGetUserMedia = null;
+
+  if (window.navigator.mozGetUserMedia) {
+    tempGetUserMedia = window.navigator.getUserMedia;
+
+    window.navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      constraints = constraints || {};
+
+      if (constraints.video ? !!constraints.video.mediaSource : false) {
```

Comment:
Cleaner to say if (constraints.video && constraints.video.mediaSource) ?
(same comment for the identical conditions bellow)